ruby_classes/game.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at, :publish_date
+
+  def initialize(multiplayer, last_played_at, publish_date)
+    super(id: nil, publish_date: publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    current_year = Time.new.year
+    elapsed_time = current_year - last_played_at
+    super() and (elapsed_time > 2)
```

Comment:
- [ ] according to [the general Ruby requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) provided by Microverse the `and` and `or` keywords are banned instead we need to use `&&` and `||`